hbase-server/src/main/java/org/apache/hadoop/hbase/util/NettyEventLoopGroupConfig.java

```diff
@@ -37,8 +38,15 @@
 /**
  * Event loop group related config.
  */
-@InterfaceAudience.Private
+@InterfaceAudience.LimitedPrivate(HBaseInterfaceAudience.CONFIG)
```

Comment:
I think IA.LP(config) means the class name is used as a config so you should not change the class name, here we just put some config names in it, I think it could be marked as IA.Private? As changing the name does not affect end users.